### PR TITLE
Feature/model 7 all risks

### PR DESCRIPTION
**Pull request by mattkappel**
- Adds all risk effects (sex, age, race/cytogenetic risk, and renal function) 
- Adds stratification by risk for survival observer
- Corrects for target coverage values for isa in the RRMM lines 
- Increases draw count to 66 from 30.

### CODE REVIEW

File: src/vivarium_csu_sanofi_multiple_myeloma/constants/data_values.py [hunk 1]
```diff
@@ -51,3 +53,86 @@
...
+# Risk Exposure Distributions for Risk Effects Calculation
+SEX_RISK_EXPOSURE = 0.539  # exposed: male
+AGE_RISK_EXPOSURE = 0.647  # exposed: 65+
+RACE_RISK_EXPOSURE = 0.177  # exposed: black
+CYTOGENETIC_RISK_EXPOSURE = 0.872  # exposed: high
+RENAL_RISK_EXPOSURE = 0.081  # exposed: impaired
+
+RACE_AND_CYTO_EXPOSURES = {
+    RISK_EXPOSURE_LEVELS.high_cytogenetic_risk_and_black: RACE_RISK_EXPOSURE * CYTOGENETIC_RISK_EXPOSURE,
+    RISK_EXPOSURE_LEVELS.high_cytogenetic_risk_and_non_black: 1 - RACE_RISK_EXPOSURE * CYTOGENETIC_RISK_EXPOSURE,
```
Comment (aflaxman) [inline src/vivarium_csu_sanofi_multiple_myeloma/constants/data_values.py:104]: Perhaps should this be `(1 - RACE_RISK_EXPOSURE) * CYTOGENETIC_RISK_EXPOSURE` ?

**Review by aflaxman — APPROVED** [review]

File: src/vivarium_csu_sanofi_multiple_myeloma/constants/data_values.py (hunk 1 above)
Comment (yongqx) [reply src/vivarium_csu_sanofi_multiple_myeloma/constants/data_values.py:104]: Yes, i think the bracket is missing here.

File: src/vivarium_csu_sanofi_multiple_myeloma/components/observers.py [hunk 2]
```diff
@@ -298,56 +300,62 @@ def on_collect_metrics(self, event: 'Event') -> None:
...
-                              .rename('right_censored'))
-            treatment_line = current_state.split('_')[-1]
-            for interval, count in right_censored.iteritems():
-                key = self.sim_end_template.format(
-                    treatment_line=treatment_line,
-                    period_end=interval.right,
+            for risk_status in itertools.product(*data_values.RISK_LEVEL_MAP.values()):
+                denominator = self.subset_risk_denominator(pop, risk_status)
+                denominator = self.subset_state_denominator(denominator, current_state, next_state, event)
+                right_censored_mask = ~(
```
Comment (KjellSwedin) [inline src/vivarium_csu_sanofi_multiple_myeloma/components/observers.py:342]: Is the intent to overwrite/refine `denominator`?

**Review by KjellSwedin — APPROVED** [review]

**Comment by mattkappel** [comment]
Note that this PR has updated the retreatment algorithm per @alibow vivarium_research PR #549

File: src/vivarium_csu_sanofi_multiple_myeloma/components/observers.py (hunk 2 above)
Comment (mattkappel) [reply src/vivarium_csu_sanofi_multiple_myeloma/components/observers.py:342]: Yessir!

**Pull request merged**